oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandlerTest.java

```diff
@@ -158,6 +158,8 @@ void shouldRedirectToRpRedirectUriWithCodeAndStateOnSuccessfulTokenResponse()
 
         when(authorizationService.validateRequest(any(), any())).thenReturn(true);
         when(tokenService.sendTokenRequest(any())).thenReturn(SUCCESSFUL_TOKEN_RESPONSE);
+        when(accountInterventionService.getAccountState(anyString(), any()))
+                .thenReturn(new AccountInterventionState(false, false, false, false));
```

Comment:
I'm curious, why did this need to be added?

Reply:
Without it `AccountInterventionStatus accountStatus = accountInterventionState.getStatus()` throws an error because `accountInterventionState` is `null`. This line mocks `noInterventionResponse()` from `AccountInterventionState::getAccountState`. Previously we were just returning `null` which didn't matter as `ACCOUNT_INTERVENTION_SERVICE_ACTION_ENABLED = false` for this test